boards/arm/s32z270dc2_r52/doc/index.rst

```diff
@@ -139,6 +140,14 @@ which supports CAN 2.0 and CAN FD protocol.
 CAN driver supports classic (CAN 2.0) and CAN FD mode. Remote transmission request is
 not supported as this feature is not available on NXP S32 CANXL HAL.
 
+Improved Inter-Integrated Circuit (I3C)
+=======================================
+
+The I3C module is intended to operate in either I3C or legacy I2C protocol modes. Due
```

Comment:
can we format this as a `.. warning::`? this is not something minor and it could be overlooked by users.